planner/core/cache.go

```diff
@@ -70,12 +70,14 @@ type pstmtPlanCacheKey struct {
 
 // Hash implements Key interface.
 func (key *pstmtPlanCacheKey) Hash() []byte {
```

Comment:
By the way, how about refactor kv.SimpleCache use []byte as key ? (not in this pr)
I don't see any benefit of its Key definition. @lysu